x-pack/plugins/cloud_security_posture/public/pages/rules/rules_table_header.tsx

```diff
@@ -24,6 +32,7 @@ interface RulesTableToolbarProps {
   selectedRulesCount: number;
   searchValue: string;
   isSearching: boolean;
+  lastModified?: string;
```

Comment:
so i did this because we display: "Last modifications to integration at X" 

and we don't want to render the text at all if we don't have `lastModified`, which is possible since we extract it from results, so if there aren't any, this shouldn't be displayed